pmm-app/src/pmm-check/components/FailedChecksTab/FailedChecksTab.utils.test.ts

```diff
@@ -0,0 +1,39 @@
+import { saveShowSilencedValue, loadShowSilencedValue } from './FailedChecksTab.utils';
+import { SHOW_SILENCED_VALUE_KEY, SHOW_SILENCED_DEFAULT } from './FailedChecksTab.constants';
+
+let getItemSpy: jest.SpyInstance;
+let setItemSpy: jest.SpyInstance;
+
+describe('FailedChecksTab::utils', () => {
+  beforeEach(() => {
+    getItemSpy = jest.spyOn(Storage.prototype, 'getItem').mockImplementation(() => 'true');
+    setItemSpy = jest.spyOn(Storage.prototype, 'setItem');
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  test('saveShowSilencedValue calls localStorage.setItem', () => {
+    saveShowSilencedValue(true);
+
+    expect(setItemSpy).toBeCalledTimes(1);
+    expect(setItemSpy).toBeCalledWith(SHOW_SILENCED_VALUE_KEY, 'true');
+  });
+
+  test('loadShowSilencedValue calls localStorage.getItem', () => {
+    jest.spyOn(Storage.prototype, 'getItem').mockImplementation(() => 'true');
```

Comment:
Instead of repeating the whole expression on every override, maybe it's easier to:
```
let getItemSpy = jest.spyOn(Storage.prototype, 'getItem');
let setItemSpy = jest.spyOn(Storage.prototype, 'setItem');
// ...
beforeEach(() => {
  getItemSpy.mockImplementation(() => 'true');
});
// ...
test('test', () => {
 // getItemSpy.mockImplementation(...)
});
```

Reply:
And this mock in particular is already in `beforeEach`

Reply:
Test updated